### ISSUE TITLE
Temperature setting issues and change reduced frame alignment suggestion

### ISSUE THREAD

**Issue by MrPerruche** [ThatTonybo/BeamAdvisor#3]
**Temperature setting issues**
When it is set on Farenheight it display celcius. When it is set on Celcius, it will turn celcius (which the mod thinks it is farenheight) and turn it into whatever the mod thinks it is celcius.

**Change reduced frame alignment suggestion**
Reduced UI looks bad in bottom corners. Here is how it currently looks (and when "top" setting is enabled if added)
![Img 1](https://user-images.githubusercontent.com/60526065/164536243-c566159b-24e3-4f50-a73b-69d17a9230b4.JPG)
With new option, set to "bottom" it will looks like this (Although even lower)
![dsqdqsddqsdaaabbbbbb](https://user-images.githubusercontent.com/60526065/164536608-51cd5f66-7c7f-4c33-861c-de6e79f31b11.JPG)

**Comment by ThatTonybo** [comment]
Frame alignment is intentional - it's how it works in ETS2. It simply trims off the bottom area.

I'll take a look into the temp unit and see if something's happening.

**Comment by O1LER** [comment]
To fix temp calculation, change the fahrenheit to celcius calculation around:
![image](https://user-images.githubusercontent.com/44237025/176223552-6b71bff9-60ba-4854-90d1-89381c305e3f.png)
To this:
![image](https://user-images.githubusercontent.com/44237025/176223592-bbc302d2-e601-4e89-b996-de43d15e65e6.png)

**Comment by ThatTonybo** [comment]
Does BeamNG provide temperatures in Celcius always?

**Comment by O1LER** [comment]
You are right, it changes with what the user sets in the options. So as long as the game is set to °F your calculation is correct, when its set to °C mine is. So theres got to be another solution

**Comment by ThatTonybo** [comment]
Okay, that makes sense. Changing units in the settings does break it. I'll see if there's some kind of way to work out what units the player has selected, so I can try fix the temperature issue. I'm keeping that new code in mind though and will use it once I figure the units out. Thanks for putting it through.
